custom/icds_reports/middleware.py

```diff
@@ -31,6 +31,9 @@
 # after split we get ['', 'a', 'domain', 'icds-dashboard-view-name', 'sub-view']
 def is_path_in_audit_urls(request):
     path = getattr(request, 'path', '').split('/')
+    # special case if a path contains logout it's ignored
+    if 'logout' in path:
```

Comment:
if you dont want to apply audit on this. good to add the URL name in `exclude_urls` above that would handle it

Reply:
I added that initially, but AUDIT_URLS check for path[3] as well as path[4] and log out can appear in any of these so it's getting ignored because it's appended to `icds_dashboard_mobile` which allowed the same.